scripts/helper.py

```diff
@@ -138,6 +138,6 @@ def parse(l):
 
 def update_config_with_sector_opts(config, sector_opts):
     for o in sector_opts.split("-"):
-        if o.startswith("CF:"):
+        if o.startswith("CF+"):
```

Comment:
This was the only thing I was wondering about

Reply:
See the discussion at https://github.com/PyPSA/pypsa-eur-sec/pull/251. 

This should have been "+" already.